gems/audited/GHSA-hjp3-5g2q-7jww.yml

```diff
@@ -0,0 +1,37 @@
+---
+gem: audited
+ghsa: hjp3-5g2q-7jww
+url: https://github.com/collectiveidea/audited/security/advisories/GHSA-hjp3-5g2q-7jww
+title: Race Condition leading to logging errors
+date: 2023-05-01
+description: |
+  'In certain setups with threaded web servers, Audited's use of
+  `Thread.current` can incorrectly attributed audits to the wrong user.
+
+  Fixed in 5.3.3.
+
+  In March, @convisoappsec noticed that the library in question had a
+  Race Condition problem, which caused logs to be registered at times
+  with different users than those who performed the genuine actions.
+
+  - The first issue we identified was from November 2021
+    - https://github.com/collectiveidea/audited/issues/601
+
+  - So the solution was implemented in the following Pull Request
+    - https://github.com/collectiveidea/audited/pull/669
+
+  - And the feature was published in version 5.3.3
+    - RELEASE: https://github.com/collectiveidea/audited/pull/671'
+cvss_v3: 3.1
+unaffected_versions:
+  - '< 4.0.0'
+patched_versions:
+  - '>= 5.3.3'
+related:
+  url:
+    - https://github.com/collectiveidea/audited/security/advisories/GHSA-hjp3-5g2q-7jww
+    - https://github.com/collectiveidea/audited/issues/601
+    - https://github.com/collectiveidea/audited/pull/669
+    - https://github.com/collectiveidea/audited/pull/671
+    - https://github.com/advisories/GHSA-hjp3-5g2q-7jww
+notes: 'vulnerableVersionRange: ">= 4.0.0, < 5.3.3";   firstPatchedVersion: 5.3.3'
```

Comment:
`notes:` should be human readable text, also this information is already captured in `patched_versions` and `unaffected_versions`. I think it's safe to omit `notes:` from these advisories.

Reply:
README says "internal notes" - I'm showing the before text before I manually converted it to your format.

Reply:
fixed